projects/packages/my-jetpack/_inc/data/types.ts

```diff
@@ -68,3 +68,21 @@ export type Purchase = {
 	total_refund_text: string;
 	check_dns: boolean;
 };
+
+export type BackupRewindableEvent = {
```

Comment:
There's a lot more to this type

```json
{
    "last_rewindable_event": {
        "summary": "Image uploaded",
        "content": {
            "text": "Jetpack-Awareness-Video-2024-full-thumbnail.jpg",
            "ranges": [
                {
                    "url": "https:\/\/wordpress.com\/media\/230032379\/10",
                    "indices": [
                        0,
                        47
                    ],
                    "id": 10,
                    "parent": null,
                    "type": "a",
                    "site_id": 999,
                    "section": "media",
                    "intent": "edit",
                    "context": "single"
                }
            ]
        },
        "name": "attachment__uploaded",
        "actor": {
            "type": "Person",
            "name": "demo",
            "external_user_id": 1,
            "wpcom_user_id": 999,
            "icon": {
                "type": "Image",
                "url": "https:\/\/secure.gravatar.com\/avatar\/880999ccfe05ecd2031b8a4b6b2958e4?s=96&d=mm&r=g",
                "width": 96,
                "height": 96
            },
            "role": "administrator"
        },
        "type": "Announce",
        "published": "2024-02-28T21:36:47.575+00:00",
        "generator": {
            "jetpack_version": 13.2,
            "blog_id": 230032379
        },
        "is_rewindable": true,
        "rewind_id": "1709156206.1545",
        "gridicon": "image",
        "status": "success",
        "activity_id": "7Kqm8Y0BXy7EB2sbGCuZ",
        "image": {
            "available": true,
            "type": "Image",
            "name": "Jetpack-Awareness-Video-2024-full-thumbnail.jpg",
            "url": "https:\/\/i0.wp.com\/mysteriously-fried.jurassic.ninja\/wp-content\/uploads\/2024\/02\/Jetpack-Awareness-Video-2024-full-thumbnail.jpg?ssl=1",
            "thumbnail_url": "https:\/\/i0.wp.com\/mysteriously-fried.jurassic.ninja\/wp-content\/uploads\/2024\/02\/Jetpack-Awareness-Video-2024-full-thumbnail.jpg?fit=96%2C96&#038;ssl=1",
            "medium_url": "https:\/\/i0.wp.com\/mysteriously-fried.jurassic.ninja\/wp-content\/uploads\/2024\/02\/Jetpack-Awareness-Video-2024-full-thumbnail.jpg?w=846&#038;ssl=1"
        },
        "is_discarded": false
    },
    "undo_backup_id": "1709155962.8212"
}
```

but I'm not sure it's necessary to document here as we don't use it all. It might be nice to have 🤷🏻

Reply:
Tbh, these types are not even really used currently as `useSimpleQuery()` is invoked in *.js files 😅 We can grow it more as we go.